changelogs/fragments/final_countdown.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
```

Comment:
```suggestion
major_changes:
```
Calling that "minor" is quite an under-statement ;-)

Reply:
OK.